samples/net/cloud/aws_iot_mqtt/src/dhcp.h

```diff
@@ -0,0 +1,14 @@
+/* DHCPv4 client startup. */
+
+/*
+ * Copyright (c) 2018 Linaro Ltd
```

Comment:
The copyright seems wrong.

Reply:
Sounds fine.

Reply:
I made the changes and updated the license in `dhcp.h` and `dhcp.c`